### PR TITLE
feat: Add a metric to measure the tile count of adm response

### PR DESCRIPTION
**Pull request by ncloudioj**
This closes [DISCO-2346](https://mozilla-hub.atlassian.net/browse/DISCO-2346).

r? @pjenvey for extra eyeballs since this is the first time to use a distribution metric in Contile.

[DISCO-2346]: https://mozilla-hub.atlassian.net/browse/DISCO-2346?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ncloudioj** [comment]
We could either increment a counter such as `contile.tiles.adm.response` with a new tag `tile.count`, but that will add this tag to all the subsequent metrics which would be wasteful, or use a plain counter `contile.tiles.adm.response.tile_count`, but that'll be not super easy to use for monitoring & analysis.

File: src/adm/tiles.rs [hunk 1]
```diff
@@ -245,6 +245,12 @@ pub async fn get_tiles(
                 })?
         }
     };
+    metrics.distribution_with_tags(
+        "tiles.adm.response.distribution",
```
Comment (pjenvey) [inline src/adm/tiles.rs:249]: I don't care if 'distribution' is included in this name but maybe something more specific:
```suggestion
        "tiles.adm.response.tiles_count",
```

**Review by pjenvey — COMMENTED** [review]
@ncloudioj FYI the distribution type isn't supported by telegraf's default configuration, however it can parse them when both these are set: `datadog_extensions = true` (which we already have) and `datadog_distributions = true` (which we don't).

So this will require to SRE add that.

**Comment by ncloudioj** [comment]
Thanks, Phil! I will double check with SRE to see if those changes are viable. If not, will reformat this to other metric types.

**Comment by ncloudioj** [comment]
Per SRE, we're using a pretty old version of InfluxDB so its support of the distribution metric type is unclear. Also, this work is due on May 5th, so let's use a counter metric for this for now.

**Review by pjenvey — APPROVED** [review]
Count obviously isn't ideal here obviously but I guess we can at least look at the breakdown of counts over small intervals tagged by e.g. geo country code.

**Pull request merged**